### PR TITLE
Feature/random message gen

### PR DESCRIPTION
**Pull request by Huubvr**

### CODE REVIEW

File: pkg/api/apitest/apitest.go [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+	return &turtstate
+}
+
+func RandomBool() *bool {
+	val := rand.Int()%2 == 0
+	return &val
+}
+
+func Must(v interface{}, err error) interface{} {
```
Comment (Desahri) [inline pkg/api/apitest/apitest.go]: Please make a comment describing this function or make it a private function.

**Comment by Desahri** [comment]
Squash your first 2 commits

File: pkg/api/apitest/apitest.go [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+}
+
+func RandomMessageType() *api.MessageType {
+	vals := []api.MessageType{api.MessageTypeState, api.MessageTypePing, api.MessageTypeHandshake}
+	return &vals[rand.Intn(len(vals))]
+}
+
+func RandomHandshake() *api.Handshake {
+	//TODO: Proper way of creating handshakes
+	vers := string(rand.Intn(10)) + "." + string(rand.Intn(10)) + "." + string(rand.Intn(10))
```
Comment (ieperen3039) [inline pkg/api/apitest/apitest.go]: is random version a good idea?

Comment (helaan) [reply pkg/api/apitest/apitest.go]: TRC revisions are a number, so I suggest adhering to something like [TenVer](http://tenver.drone-ver.org/), or just return a random number between 0..99999

Comment (rvolosatovs) [reply pkg/api/apitest/apitest.go]: It's not extensible and is not what we defined in the spec

**Pull request merged**